src/Features/CSharp/Portable/CodeRefactorings/InlineTemporary/InlineTemporaryCodeRefactoringProvider.cs

```diff
@@ -69,7 +69,7 @@ internal partial class InlineTemporaryCodeRefactoringProvider : CodeRefactoringP
                 return;
             }
 
-            if (variableDeclarator.Initializer.Kind() == SyntaxKind.RefExpression)
+            if (variableDeclaration.Type.Kind() == SyntaxKind.RefType)
             {
                 // TODO: inlining byref temps is NYI
```

Comment:
I understand your PR properly disables it, but we should ensure we still have something to track. @Pilchie or @kuhlenh are we tracking this with a bug somewhere we can link to?